src/main/java/net/minecraftforge/common/ForgeConfigSpec.java

```diff
@@ -260,12 +244,9 @@ else if (dryRun)
         return count;
     }
 
-    private boolean stringsMatchIgnoringNewlines(@Nullable Object obj1, @Nullable Object obj2)
-    {
-        if (obj1 instanceof String string1 && obj2 instanceof String string2)
-        {
-            if (!string1.isEmpty() && !string2.isEmpty())
-            {
+    private boolean stringsMatchIgnoringNewlines(@Nullable Object obj1, @Nullable Object obj2) {
+        if (obj1 instanceof String string1 && obj2 instanceof String string2) {
```

Comment:
In your PR branch, the only usages I can find of this method always call it with String parameters, so we can simplify this...
```suggestion
    private boolean stringsMatchIgnoringNewlines(@Nullable String string1, @Nullable String string2) {
        if (string1 != null && string2 != null) {
```